mods/ra/maps/allies-09a/rules.yaml

```diff
@@ -0,0 +1,133 @@
+Player:
+	PlayerResources:
+		DefaultCash: 10000
+
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, allies09a.lua, allies09a-AI.lua
+	MissionData:
+		Briefing: One of Stalin's top atomic strategists, Vladimir Kosygin, wishes to defect. \n\nHis knowledge of Stalin's atomic strategies is invaluable to us. \n\nWe will extract him from the Riga compound where he is stationed. \n\nUse a spy to infiltrate the Soviet command center and contact Kosygin. \n\nOnce he is out of the building, guide him back to your base any way you can. \n
```

Comment:
Style nit: No whitespaces before line breaks (`\n`).

Reply:
We should also remove some of the new lines to be consistent with the other briefings. I suggest:
> One of Stalin's top atomic strategists, Vladimir Kosygin, wishes to defect. His knowledge of Stalin's atomic strategies is invaluable to us. We will extract him from the Riga compound where he is stationed.\n\nUse a spy to infiltrate the Soviet command center and contact Kosygin. Once he is out of the building, guide him back to your base any way you can.

Reply:
Fixed at [c9bbea5](https://github.com/OpenRA/OpenRA/commit/c9bbea5b361b63990c5c073f6ac271692cc2ea62)